src/variant.h

```diff
@@ -141,6 +141,7 @@ struct Variant {
   int extinctionOpponentPieceCount = 0;
   PieceType flagPiece = NO_PIECE_TYPE;
   Bitboard flagRegion[COLOR_NB] = {};
+  int numFlag = 1;
```

Comment:
Similar attributes are named ...Count or ...PieceCount, so for consistency I would suggest to use a similar naming here (e.g., flagCount or flagPieceCount).